reactive/kubernetes_node_base.py

```diff
@@ -92,7 +89,7 @@ def process_snapd_timer():
         leader_set({"snapd_refresh": timer})
 
 
-@when_any(KUBERNETES_CP + ".snaps.installed", "kubernetes-worker.snaps.installed")
+@when_any("kubernetes-node.snaps.installed", "kubernetes-worker.snaps.installed")
```

Comment:
The requires https://github.com/charmed-kubernetes/charm-kubernetes-control-plane/pull/208 to be merged first.

Reply:
No it doesn't.  k-c-p doesn't use this layer yet.

Reply:
No it doesn't.  k-c-p doesn't use this layer yet.